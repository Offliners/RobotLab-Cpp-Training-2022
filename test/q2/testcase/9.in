100
festival
fulfill
villa
MERiT
IndUSTRIAL
sPOT.
tOUch
trembLe
ShoRTen
PARADISE
EDUCATiONAl
LIghTEN
hawk
CoURTESY
ImpArtial
desIGnate
calculate
DEmONsTRATION
CasHiEr
DaTa
acquire
neither
MAIl
past
ChecKUp
CoMPAsS
paradoxically
sPiNAcH
PRoTESt
pUBLISH
SaLVaTiON
SiLly
WAITER
HuManity
SHOP
angeL
outdoor
OVERSEAS
cIRcuLate
KNIFE
goodS
PESSIMiSM
sURpRISed
COINcIDENCe
staTe
COMMITTEE
cola
pOOr
FAntasy
ENdORSE
STRATEGIC
LENGTH
rOAcH
COaL
oRIenTal
AD
hear
baSed
MIstake
glaSseS
TREnd
pRECAUTIon
OffiCiaL
DAddY/DAd
ECCENTRIC
ALtOGETheR
dORMitory/DoRM
HOMe
aspiRing
SCOlD
BaCKwArd
WhAt
COUnsEL
bEijIng
mEdal
tHird
act
TAnk
toMB
bReathTakiNg
rEPreseNTaTIVE
FaCTORY
raNgE
bEt
ROUgHLY
researcheR
BasiN
COnVErt
PaTTErN
usefulness
chin
hYphen
CoMEDian
tROUBLE
gRAvEsToNE
EStIMATE
regArdiNG
cEREAL
ExTREMe
nInEteEn
