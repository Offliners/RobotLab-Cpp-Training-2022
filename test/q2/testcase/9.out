n. 節日；歡樂
v. (n)實現
n. 別墅
n. 價值；長處
adj. 工業的
n./v. 地點；散布
n./v. 觸覺；觸摸
n./v. 發抖
v. 使縮短
n. 天堂
adj. 有教育意義的
v. 照亮
n. 鷹
n. 有禮的舉止
adj. 公平的
v. 指定；任命
v. 計算
n. 驗證；示威運動
n. 出納員
n. 資料
v. 取得
adj./adv./pron./conj.  兩者都不
n./v. 郵件；郵寄
adj./prep. 過去的
n. 檢查
n. 指南針；圓規
adv. 似是而非地
n. 菠菜
n./v. 抗議
v. 出版
n. 救星
adj. 愚蠢的
n. 侍者
n. 人性；人文學科
n. 商店
n. 天使
adj. 戶外的
adj./ad v. 來自海外的
v. 循環；流通
n. 刀
n. 貨物
n. 悲觀(主義)
adj. 感到驚奇的
n. 巧合
n. 狀態
n. 委員會
n. 可樂類飲料
adj. 貧窮的
n. 空想；幻想
v. 贊同
adj. 戰略(性)的
n. 長度；期間
n. 蟑鄉；大麻煙卷的煙蒂
n. 煤礦；煤塊
n./adj. 東方人；東方的
n. 廣告
v. 聽見
adj. 以…為基礎的
n. 錯誤；過失
n. 眼鏡
n. 趨勢
n. 預防措施
adj./n. 官方的；官員
n. 爹地
adj./n. 古怪的(人)
adv. 完全；總而言之
n. 學生宿舍
n./adv. 住家；家庭
adj. 有抱負的
v. 責備
adv. 反方向地
pron./conj. 什麼
n. 忠告
n./prop. 北京
n. 獎章；紀念章
adj./n. 第三
n./v. 行動；表演
n. 大容器
n. 墳
adj. 使人興奮的
adj./n. 象徵的；例子
n. 工廠
n. 區域
n./v. 打賭
adv. 大體上
n. 研究員
n. 臉盆；洗滌槽
v. 轉換
n. 樣式
n. 有效
n. 下巴
n. 連字號
n. 丑角式人物
n./v. 憂慮
n. 墓碑
n./v. 估計；預計
prep. 關於…
n. 玉蜀黍薄片
adj./n. 盡頭的；極限
adj./n. 十九
